### ISSUE TITLE
BUG: Messages not showing up in Inbox (for recipient) or Sent box (for sender)

### ISSUE THREAD

**Issue by klohman** [ustaxcourt/ef-cms#7381]
**Describe the Bug**
After sending a message, the message is not displaying in the recipient's inbox or the sender's Sent box. This is not happening consistently. [NOTE: Likely related to ES issue]

**Business Impact/Reason for Severity**
Court uses the messaging function to move work around the Court -- critical to Court operations.

**In which environment did you see this bug?**
stg

**Who were you logged in as?**
docket clerk 1
petitions clerk 1

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
manual testing 

**To Reproduce**
1. Go to case
2. Click Create and Message 
3. Send message 
4. View Sent box 
5. Login as recipient 
6. View Inbox 

**Expected Behavior**
Message should display in recipient's Inbox (and Section Inbox) and in the sender's Sent box (and Section Sent box). 

**Actual Behavior**
Message is not displaying in Inbox or Sent box. Message is displaying in Case Messages. 

**Screenshots**

https://drive.google.com/file/d/1qMwvmAHggfXpXxrqBrTm91FOB5vIBuck/view?usp=sharing

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

**FOR ENGINEERING TEAM ONLY**
Bug Resolution Steps:
 - [ ] Determine why the bug wasn't caught by a test. 
 - [ ] Determine if an automated test needs to fixed, expanded or created. If unsure, bring in others to discuss.
 - [ ] Determine if a manual test needs to be fixed, expanded or created. If unsure, bring in others to discuss.
 - [ ] If needed, automated test is created.
 - [ ] If needed, manual test is created.
 - [ ] Reason for bug has been documented.
 - [ ] Fix has been deployed to dev environment.
 - [ ] Fix has been deployed to the stage environment.
 - [ ] Bug has been tested in staging (UX or Engineering).

**Comment by rachelschneiderman** [comment]
Unable to reproduce locally or on dev. Seems likely this is an elasticsearch issue. Will try to recreate on prod once migration has completed (on 11/2).

**Comment by rachelschneiderman** [comment]
We were unable to reproduce the issue on local and dev, we were able to see the problem in prod. We noticed the elasticsearch instance for prod has `0` indexed documents, which is most likely the cause of the issue. We also noticed there is an active migration happening on the prod environment which is causing a delay to the processStreamRecords lambda. We are blocked from further investigation until the ongoing data import on prod has been completed.
